packages/camera/camera/ios/Classes/camera-umbrella.h

```diff
@@ -8,6 +8,7 @@
 #import <camera/FLTThreadSafeFlutterResult.h>
 #import <camera/FLTThreadSafeMethodChannel.h>
 #import <camera/FLTThreadSafeTextureRegistry.h>
+#import <camera/FlashMode.h>
```

Comment:
```suggestion
```

Looks like this is only needs to be exposed to the plugin implementation and a test, and we don't want these symbols exported to the wider Flutter app, or other plugins.
Fortunately most of the work for a `Test` module work was done in https://github.com/flutter/plugins/pull/4430 (which was cherry-picked with https://github.com/flutter/plugins/pull/4426#discussion_r727762574).

I think adding something like this to the modulemap should do it:
https://github.com/flutter/plugins/blob/d20d3bf48deb5432dfc44c8ae0debab0626e7830/packages/camera/camera/ios/Classes/CameraPlugin.modulemap#L7-L9
```diff
  explicit module Test {
    header "CameraPlugin_Test.h"
+   header "FlashMode.h"
  }
```

Then module imported in the test like this:
https://github.com/flutter/plugins/blob/d20d3bf48deb5432dfc44c8ae0debab0626e7830/packages/camera/camera/example/ios/RunnerTests/CameraOrientationTests.m#L6

And the header directly imported in the `CameraPlugin.m` implementation as you already have it.

See https://github.com/flutter/plugins/pull/4157 for more context about CocoaPods and modulemaps.

Reply:
sounds reasonable. will take a look at how that works.

Reply:
The modulemap is very convenient!

Reply:
Fortunately there was already one there.  Trust me it was really annoying to figure out how to make it work with CocoaPods the first time I did this. https://github.com/flutter/plugins/pull/4157 🙂